src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,85 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.NotNull;
+
+public class Calculator {
+    int a = Integer.MIN_VALUE;
+    int b = Integer.MIN_VALUE;
+
+    String operand = "";
+
+    public int add(int a, int b) {
+        return a+b;
+    }
+    public int substract(int a, int b) {
+        return a-b;
+    }
+    public int multiply(int a, int b) {
+        return a*b;
+    }
+    public int divide(int a, int b) {
+        return a/b;
+    }
+
+    public static void main(String[] args) {
+        String input = "2 + 3 / 1";
+        String[] numbers = input.split("[^0-9]");
+        String[] operands = input.split("[0-9]+");
+        for(String s : numbers) System.out.print(s);
+        System.out.println("");
+        for(String s : operands) System.out.print(s);
```

Comment:
한줄이라도 중괄호를 생략하지 않는 것이 컨벤션입니다 :) 

```suggestion
        for(String s : numbers) {
            System.out.print(s);
        }

        for(String s : operands) {
            System.out.print(s);
        }
```

의미 없는 출력은 제거해 보는 것이 어떨까요?